pephub/routers/api/v1/namespace.py

```diff
@@ -63,13 +63,17 @@ async def get_namespace_projects(
         search_result = agent.annotation.get(
             namespace=namespace, limit=limit, offset=offset, admin=namespace_access
         )
+    results = [p.dict() for p in search_result.results]
+
+    # sort by last_update_date
```

Comment:
1) if action in next line is explicit, we shouldn't add any comments. I think sorting function is understandable:)
2) This functionality should be in pepdbagent, not here. I wiil create Issue on this topic.